### DISCUSSION TITLE
TURN example work just once

### DISCUSSION THREAD

**Discussion by Ali-Mirghasemi** [webrtc-rs/webrtc#59] [Q&A]
hello 
when i run TURN example it's works just once
turn args
```
--port 3478 --public-ip 192.168.1.250 --users user=pass
```

and tested with 
```
https://webrtc.github.io/samples/src/content/peerconnection/trickle-ice/
```

give me valid params just first time i click gather 

*port-forwarding and NAT enabled in router and my public ip have DDNS

**Comment by rainliu** [comment]
Thanks for reporting this issue. Will take a look soon
